### PR TITLE
Landing page added in video library

### PR DESCRIPTION
**Pull request by ashifkhn**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,24 +1,23 @@
-import logo from './logo.svg';
-import './App.css';
+import {
+  BrowserRouter,
+  Routes,
+  Route,
+} from "react-router-dom";
+import { HomePage } from "./Pages/HomePage/HomePage";
+import { LandingPage } from './Pages/LandingPage/LandingPage';
+import { SignUpPage } from "./Pages/SignUpPage/SignUpPage";
```
Comment (vikrantkode) [inline src/App.js:8]: use barrel exports

File: src/Style/landingpage.scss [hunk 2]
```diff
@@ -0,0 +1,119 @@
+.login_page_container {
+  height: 100vh;
+  background-color: #fff;
+  display: flex;
+  align-items: center;
+  font-family: Gotham;
+  .image_container_login_page {
+    background-color: #0b1235;
```
Comment (vikrantkode) [inline src/Style/landingpage.scss:8]: try avoiding use of hardcode colors

**Review by vikrantkode — APPROVED** [review]
- Landing page looks good
- looking forward for more features

**Review by Pratham82 — APPROVED** [review]

**Comment by Pratham82** [comment]
Looks good, you can merge this.

**Pull request merged**